### PR TITLE
7.0 pep8

### PR DESCRIPTION
**Pull request by lmignon**
This PR follows the PR #16 and additionally to the changes from PR #16 provides the PEP8 changes required to make travis happy.

### CODE REVIEW

File: base_geoengine_demo/geo_npa.py [hunk 1]
```diff
@@ -36,98 +34,47 @@ def _get_ZIP_total_sales(self, cursor, uid, ids, name, args, context=None):
...
+
+    def name_get(self, cursor, uid, ids, context=None):
         res = []
-        for r in self.browse(cursor, uid, ids) :
-            res.append((r.id, u"%s %s" %(r.name, r.city)))
+        for r in self.browse(cursor, uid, ids):
+            res.append((r.id, u"%s %s" % (r.name, r.city)))
         return res
-
-    def test_func(self, cursor, uid, ids):
```
Comment (pedrobaeza) [inline base_geoengine_demo/geo_npa.py:82]: Maybe this can be converted to a proper test

**Comment by pedrobaeza** [comment]
Looks good, but let's wait for Travis to say it.

**Comment by pedrobaeza** [comment]
Restarted Travis due to an strange error, but pylint check was OK.

:+1:

File: base_geoengine_demo/geo_npa.py (hunk 1 above)
Comment (lmignon) [inline base_geoengine_demo/geo_npa.py:82]: You are right @pedrobaeza  I'll write the test monday.

**Comment by lmignon** [comment]
@pedrobaeza thank you for the review.

**Comment by lmignon** [comment]
@pedrobaeza The removed function `def test_func(self, cursor, uid, ids)` has been converted to proper unittests https://github.com/acsone/geospatial/commit/2ddc957008ddc71dd667ad702cdb4291e8a1092e#diff-bad79d5963506a7b6591a69803725f71R34

**Comment by pedrobaeza** [comment]
Thanks you. Very fine job!

:+1: if everything goes OK

**Comment by yvaucher** [comment]
:+1: Thanks for the work

**Comment by bwrsandman** [comment]
:+1: Ready to merge?

File: base_geoengine/doc/source/conf.py [hunk 2]
```diff
@@ -11,23 +12,29 @@
...
-#sys.path.insert(0, os.path.abspath('/opt/openerp_61Launchpad/src/server/'))
-sys.path.insert(0, os.path.abspath('/opt/openerp_61Launchpad/src/server/openerp/'))
-sys.path.insert(0, os.path.abspath('/opt/openerp_61Launchpad/src/server/openerp/addons/'))
+# sys.path.insert(0, os.path.abspath('/opt/openerp_61Launchpad/src/server/'))
+sys.path.insert(
+    0,
+    os.path.abspath('/opt/openerp_61Launchpad/src/server/openerp/'))
+sys.path.insert(
+    0,
+    os.path.abspath('/opt/openerp_61Launchpad/src/server/openerp/addons/'))
```
Comment (ovnicraft) [inline base_geoengine/doc/source/conf.py:27]: why this static path here ?

**Comment by lmignon** [comment]
@ovnicraft The documentation is not up to date on 7.0. It has been fixed on 8.0 and is published at http://oca.github.io/geospatial/

**Pull request merged**